packages/ra-ui-materialui/src/layout/Sidebar.js

```diff
@@ -11,6 +11,18 @@ export const CLOSED_DRAWER_WIDTH = 55;
 
 const useStyles = makeStyles(
     theme => ({
+        root: {},
+        docked: {},
+        paper: {},
+        paperAnchorLeft: {},
+        paperAnchorRight: {},
+        paperAnchorTop: {},
+        paperAnchorBottom: {},
+        paperAnchorDockedLeft: {},
+        paperAnchorDockedTop: {},
+        paperAnchorDockedRight: {},
+        paperAnchorDockedBottom: {},
+        modal: {},
         drawerPaper: {
```

Comment:
So now there are no differences between the `drawerPaper` and the `paper` classnames? I think we can use only the `paper` key and mark the `drawerPaper` key as deprecated.

Reply:
That's a BC break and it brings small added value. Let's merge this PR as is.